specs/commandline/inspect.md

```diff
@@ -0,0 +1,271 @@
+└──application/vnd.cncf.notary.signature
+    ├── sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+      ├──signingAlgorithm: RSASSA-PSS-SHA-256
+      ├──signedAttributes
+            ├──contentType: application/vnd.cncf.notary.payload.v1+json
+            ├──signingScheme: notary.default.x509
+            ├──signingTime: 2022-04-06T07:01:20Z
+            ├──expiry: 2022-10-06T07:01:20Z
+            ├──io.cncf.notary.verificationPlugin: com.example.nv2plugin     //extended attributes to support plugins
+      ├──userDefinedAttributes
+            ├──io.wabbit-networks.buildId: 123                              //user defined payload annotations.
+      ├──unsignedAttributes
+            ├──io.cncf.notary.timestampSignature: <Base64(TimeStampToken)>  //how TSA response (time stamp token) is represented.
+            ├──io.cncf.notary.signingAgent: notation/1.0.0                  //provides the identifier of a client (e.g. Notation) that produced the signature
+      ├──certificates
+            ├──SHA1Fingerprint: 2f1cc5b8455381cdefac83b4bd305b789cc9c16e
+              ├──issuedTo: Microsoft Root Certificate Authority 2010
+              ├──issuedBy: Microsoft Root Certificate Authority 2010
+              ├──expires: Sat Jun 23 22:04:01 2035
+            ├──SHA1Fingerprint: 8BFE3107712B3C886B1C96AAEC89984914DC9B6B
+              ├──issuedTo: Microsoft Code Signing PCA 2010
+              ├──issuedBy: Microsoft Root Certificate Authority 2010
+              ├──expires: Sun Jul 06 20:50:17 2025
+      └──payload                                                           //descriptor of the target artifact manifest that is signed.
+            ├──mediaType: application/vnd.oci.image.manifest.v1+json
+            ├──digest: sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+            └──size: 16724
+      
+    ├── sha256:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+      ├──signingAlgorithm: RSASSA-PSS-SHA-256
+      ├──signedAttributes
+            ├──contentType: application/vnd.cncf.notary.payload.v1+json
+            ├──signingScheme: notary.signingAuthority.x509
+            ├──signingTime: 2022-04-06T07:01:20Z
+            ├──expiry: 2022-10-06T07:01:20Z
+            ├──io.cncf.notary.verificationPlugin: com.example.nv2plugin                  
+      ├──unsignedAttributes
+            ├──io.cncf.notary.timestampSignature: <Base64(TimeStampToken)>
+            ├──io.cncf.notary.signingAgent: notation/1.0.0                  
+      ├──certificates
+            ├──SHA1Fingerprint: 2f1rr5b8455381frdajc83b4bd305b743cc9513u
+              ├──issuedTo: Microsoft Root Certificate Authority 2010
+              ├──issuedBy: Microsoft Root Certificate Authority 2010
+              ├──expires: Sat Jun 23 22:04:01 2035
+            ├──SHA1Fingerprint: 8BFE3107712B3C886B1C96AAEC89984914DC9B6B
```

Comment:
The format of the SHA1 fingerprint is not consistent. Should they all upper cases or lower cases?

Reply:
> The format of the SHA1 fingerprint is not consistent. Should they all upper cases or lower cases?

SHA1 is all uppercase and fingerprint should be lowercase.

Reply:
I mean the value of the `SHA1 fingerprint` field.

As you can see, the value `8BFE3107712B3C886B1C96AAEC89984914DC9B6B` are all upper cases and `2f1rr5b8455381frdajc83b4bd305b743cc9513u` are all lower cases.

Reply:
> I mean the value of the `SHA1 fingerprint` field.
> 
> As you can see, the value `8BFE3107712B3C886B1C96AAEC89984914DC9B6B` are all upper cases and `2f1rr5b8455381frdajc83b4bd305b743cc9513u` are all lower cases.

Keeping it uppercase based on the examples I am seeing.